sources/packages/forms/src/form-definitions/sfaa2025-26.json

```diff
@@ -10905,6 +10905,49 @@
               "lazyLoad": false,
               "tags": [],
               "hideLabel": true
+            },
+            {
+              "autofocus": false,
+              "input": true,
+              "tableView": false,
+              "inputType": "radio",
+              "label": "Is your parent(s)/step-parent/sponsor/legal guardian a resident of B.C.?",
```

Comment:
There is an exception associated to answering NO to this question. Is the exception obsolete? 
![image](https://github.com/user-attachments/assets/0fc92314-eee1-4d55-8d87-f1a0494497bc)